### PR TITLE
Kinesso ID: add UserId module and fixes conflicts on pr 7077

### PR DESCRIPTION
**Pull request by ChrisHuie**
This pr fixes testing conflicts on pr -> #7077 and since I don't have access to that branch if we approve and merge this pr I can delete that one.

### CODE REVIEW

File: modules/kinessoIdSystem.js [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+/**
+ * creates and logs the error message
+ * @function
+ * @param {string} error message
+ * @returns {Error}
+ */
+function createError(message) {
+  utils.logError(message);
+  const err = new Error(message);
+  err.source = 'sharedId';
```
Comment (osazos) [inline modules/kinessoIdSystem.js:116]: Should not it be `kinessoId`?

Comment (ChrisHuie) [reply modules/kinessoIdSystem.js]: Yeah 😬 fixed it

**Review by osazos — APPROVED** [review]
LGTM

**Comment by bretg** [comment]
docs pr https://github.com/prebid/prebid.github.io/pull/3215

**Pull request merged**